celestia/peers.txt

```diff
@@ -1 +1 @@
-
+62b11589e64b4cef9f7202a2fd012937b1522745@65.21.69.251:26656
```

Comment:
please fix conflict by submitting the peer info in a separate file in this directory named as `your-validator.txt`

Reply:
I resolved conflict but removing peer from peer.txt, can I submit spidey.txt peer as a separate PR?